src/meteodatalab/operators/vertical_interpolation.py

```diff
@@ -317,7 +317,7 @@ def interpolate_k2any(
     field: xr.DataArray,
     mode: Literal["low_fold", "high_fold"],
     tc_field: xr.DataArray,
-    tc_values: Sequence[float],
+    tc: TargetCoordinates,
```

Comment:
This is going to break the demo 🔨

Reply:
Should we then create a tag before merging this PR?

Reply:
why do we need to take the description of the coordinates away from the tc_field ? 
What use cases are not supported right now? 
I liked the idea of fetching the information from tc_field because it guarantees to be consistent. 
Now I can pass a tc_field that has type_of_level="pressure" and a TargetCoordinate with type_of_level echoTopInDBZ. 
I think this was wrong: 
https://github.com/MeteoSwiss/meteodata-lab/pull/27/files#diff-12e768f61f94d786b924b9a239a1d09d548c105c2dc188d988261b191342ba10L362

why should not get the type_of_level of the target from the shortName, but why not getting the type_of_level from the type_of_level of the tc_field ?

Reply:
There's nothing in `tc_field` that corresponds to the type of level of the interpolation result. This would have required the introduction of yet another mapping e.g. `DBZ` to `echoTopInDBZ` and `P` to `isobaricInPa`. Note that this approach also assumes that the only parameter with vertical coordinates in DBZ is echo top. 

The type of level of tc_field is not related to the type of level for interpolation result.